src/js/i18n/definitions.js

```diff
@@ -49,6 +49,7 @@ export const formFeedback = defineMessages({
 export const misc = defineMessages({
   airport: 'Airport',
   citySeal: 'City of Austin Seal',
+  guidePagePlaceholder: 'Guide Page placeholder image',
   workInProgress: 'Alpha.austin.gov is a work in progress.\n{projectsSiteLink}',
```

Comment:
I added the text for formatting/translations used by the `alt` attribute to be handled like `citySeal` is with this strategy.